src/runtime.js

```diff
@@ -709,8 +709,9 @@ class MultidocManager {
           if (doc.body) {
             const body = importShadowBody(
                 shadowRoot, doc.body, /* deep */ true);
-            body.classList.add('amp-shadow');
-            shadowRoot.appendChild(body);
+            // TODO (torch2424): Understand why this duplicates the body when we are creating the shadow root element
```

Comment:
I think I'm missing it here: did you say before that you restored this to here? I still see these commented out.